x-pack/platform/plugins/shared/inference/server/tasks/nl_to_esql/esql_docs/esql-byte_length.txt

```diff
@@ -0,0 +1,26 @@
+# BYTE_LENGTH
+
+This function calculates the byte length of a given string.
+
+## Syntax
+
+`BYTE_LENGTH(string)`
+
+### Parameters
+
+#### string
+
+The text string for which the byte length is to be determined. If `null` is provided, the function will return `null`.
+
+## Examples
+
+Following is an example of how BYTE_LENGTH is implemented in an ES|QL query:
```

Comment:
in `x-pack/platform/plugins/shared/inference/server/tasks/nl_to_esql/esql_docs/esql-bit_length.txt`, this preamble had a different format:

> In the below example, the `BIT_LENGTH` function is used to calculate the bit length of the 'city' field from the 'airports' index where the 'country' is "India".

Is there a reason this one is different?

```suggestion
In the below example the `BYTE_LENGTH` function is used to calculate the byte length of the 'city' field from the 'airports' index where the country is "India".
```

Also, we're using all the quotation types in these sentences - is that intentional?

Reply:
> Is there a reason this one is different?

Just different generations, the page rewriting is not as strict as I'd like it to

> We're using all the quotation types in these sentences - is that intentional?

No, but they kinda make sense 😄 `` for func name, '' for field name and "" for string values.